packages/common/src/compiler/translator.ts

```diff
@@ -193,6 +193,17 @@ const convertDbtMetricToLightdashMetric = (
             sql = metric.sql;
         }
     }
+    if (metric.filters && metric.filters.length > 0) {
+        const filterSql = metric.filters
+            .map(
+                (filter) =>
+                    // eslint-disable-next-line no-useless-escape
+                    `(\$\{TABLE\}.${filter.field} ${filter.operator} ${filter.value})`,
```

Comment:
Is the field always from the same table ?

Reply:
It's a restriction I need to document! Gonna update a bunch of the metrics docs today!